documentation/docs/configuration/manifest.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 2
+---
+
+# Deployment Manifest
+
+In order for monaco to know what to deploy, there has to be a manifest file present.
+This file provides details on what to deploy and where to deploy it.
+
+## Structure
+
+The manifest is written in yaml syntax. It has two top level keys `projects` and `environments`.
+
+### Projects
+
+All entries under the `projects` top level key specify projects to deploy by monaco. To specify
+the type of a project, one has to provide the `type` key in the project item.
+
+There are currently two types of projects:
+- simple
+- grouping
+
+#### Simple Projects
+This is the default type. All you need to provide is a `name` and `path` property.
+If no `path` property is provided, the name will be used as `path`.
+
+**Note**: It is not allowed for the name to contain either `/` nor `\`. This decision
+was made to explicitly distinquish it from filesystem paths.
+
+**Note**: Paths should always use `/` as separator!
+
+E.g.
+```yaml
+projects:
+- name: infra
+  path: shared/infrastructure
+```
+
+#### Grouping Projects
+Grouping projects offer a less boilerplate way of grouping multiple projects together.
+The difference to a simple project is, that it will load all sub-folders of a given path
```

Comment:
`grouping project` rather than `it` - more explicit and descriptive, no way to missunderstand